src/oas.md

```diff
@@ -1681,6 +1748,214 @@ application/json:
       $ref: '#/components/examples/frog-example'
 ```
 
+Alternatively, since all JSON is valid YAML, the example value can use JSON syntax within a YAML document:
+
+```yaml
+application/json:
+  schema:
+    $ref: '#/components/schemas/Pet'
+  examples:
+    cat:
+      summary: An example of a cat
+      value: {
+        "name": "Fluffy",
+        "petType": "Cat",
+        "color": "White",
+        "gender": "male",
+        "breed": "Persian"
+      }
+    dog:
+      summary: An example of a dog with a cat's name
+      value: {
+        "name": "Puma",
+        "petType": "Dog",
+        "color": "Black",
+        "gender": "Female",
+        "breed": "Mixed"
+      }
+    frog:
+      $ref: '#/components/examples/frog-example'
+```
+
+###### Sequential JSON
+
+For any [sequential media type](#sequential-media-types) where the items in the sequence are JSON values, no conversion of each value is required.
+JSON Text Sequences ([[?RFC7464]] `application/json-seq` and [[?RFC8091]] the `+json-seq` structured suffix), [JSON Lines](https://jsonlines.org/) (`application/jsonl`), and [NDJSON](https://github.com/ndjson/ndjson-spec) (`application/x-ndjson`) are all in this category.
+Note that the media types for JSON Lines and NDJSON are not registered with the IANA, but are in common use.
+
+The following example shows Media Type Objects for both streaming log entries and returning a fixed-length set in response to a query.
+This shows the relationship between `schema` and `itemSchema`, and when to use each even though the `examples` field is the same either way.
+
+```YAML
+components:
+  schemas:
+    LogEntry:
+      type: object
+      properties:
+        timestamp:
+          type: string
+          format: date-time
+        level:
+          type: integer
+          minimum: 0
+        message:
+          type: string
+    Log:
+      type: array
+      items:
+        $ref: "#/components/schemas/LogEntry"
+      maxItems: 100
+  examples:
+    LogJSONSeq:
+      summary: Log entries in application/json-seq
+      # JSON Text Sequences require an unprintable character
+      # that cannot be escaped in a YAML string, and therefore
+      # must be placed in an external document shown below
```

Comment:
What's wrong with `\x1E` `\u001E`?

Reply:
Oh... those don't work in _multiline_ strings. But we don't have to use multiline strings, do we?

Reply:
Those [don't work in block scalars, only in double-quoted strings](https://yaml.org/spec/1.2-old/spec.html#id2787109).  And making that example a one-line double-quoted string would be unreadable and useless as an example.

Reply:
@hudlow (I know, I couldn't' believe it either)

Reply:
Empirically, this also seems to work but I'm not sure it's better than the single-line treatment. 😬 

```yaml
example: "\x1E{ \
    \"timestamp\": \"1985-04-12T23:20:50.52Z\", \
    \"level\": 1, \
    \"message\": \"Hi!\" \
  }\x1E{\
    \"timestamp\": \"1985-04-12T23:20:51.37Z\", \
    \"level\": 1, \
    \"message\": \"Bye!\" \
  }"
```

Reply:
@hudlow I applaud your creativity and dedication!  That said... I'm gonna go with "nope" on this one.  But it's an apperciative "nope"!

A bit more seriously, I really like having this show both inline (`value`) and external (`externalValue`) Example Objects.  And I really like being able to apply `application/json-seq`-specific syntax highlighting to help people understand what is going on with the unprintable character, which is not possible to do in an embedded YAML string (of any format).  Or at least it is way beyond my highlightjs skills.